### ISSUE TITLE
Click bug with Dragon NaturallySpeaking 

### ISSUE THREAD

**Issue by lizcameron** [bbc/simorgh#4646]
**Describe the bug**
Dragon NaturallySpeaking doesn't work as expected on the media block, article page or front page. It can navigate through the page but does not respond to clicking on links or buttons, e.g. play. Mixed results have been found when testing this, with some users able to click play but unable to then control further, e.g. pause or stop. More investigation is needed to determine if there is a problem with Simorgh pages more widely, or just a specific block/page, and what the problem is. 

**To Reproduce**
Steps to reproduce the behaviour:

1. Open Dragon NaturallySpeaking and set up (if required)
2. Open IE
3. Go to https://www.test.bbc.co.uk/news/articles/cn7k01xp8kxo
4. Use Dragon NaturallySpeaking to locate the media block
5. Attempt to play the video in the media block(Following various commands)
6. See error

**Expected behaviour**
Outlined here https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-dragon

**Screenshots**

**Desktop (please complete the following information):**

- OS: Windows 7
- Browser: IE 11
- v13.0

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

**Comment by HarveyPeachey** [comment]
Here are my findings:

The commands for Dragon Naturally Speaking vary. To play a video on our media player:
1. Tab to play button.
2. Say "Click play button".
3. See video playing.

To pause and play the video again:
1. Make sure controls are visible
2. Say "Click play"
3. Make sure controls are still visible
4. Say "Click pause"

The reason why you can't pause or play the video when the controls aren't showing is because you can't do it with normal mouse interaction either without having the controls open. This doesn't seem to be a bug but It might be confusing to users when they have to say "Click play button" to activate the first instance of media playing and then have to say "Click play/pause" to do it again.

Following on from discussion as well, links are clickable with Dragon in simorgh. To invoke a click of a link you have to say "click" followed by the name of the link e.g. To click the BBC news branding logo link you have to say "Click BBC News" and then it will bring you to the BBC news homepage. This is the same behaviour as the PAL live news site.

**Comment by HarveyPeachey** [comment]
CC: @greenc05
